app/models/campaign.rb

```diff
@@ -103,63 +107,53 @@ def test_victim
 
   private
 
-  def parse_email_addresses
-    if not self.emails.blank?
-      entry = self.emails.split("\r\n")[0]
-      if entry.scan(/,/).count == 0
-        # email
-        parse_single_csv
-      elsif entry.scan(/,/).count == 1
-        # firstname, email
-        parse_double_csv
-      elsif entry.scan(/,/).count == 2
-        # firstname, lastname, email
-        parse_triple_csv
+  def validate_email_addresses
```

Comment:
I think this method is being a bit restrictive... I'm unable to add any targets on this branch.